charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -208,6 +208,12 @@ data:
         sccEnabled: true
       {{- end }}
 
+      {{- if and .Values.global.extraAnnotations ( not .Values.global.authSidecar.enabled ) }}
```

Comment:
why does `not .Values.global.authSidecar.enabled` have to be here?

Reply:
when authsidecar is enabled passing extra ingress annotation back to houston code does get respected thats why we added this additional condition

Reply:
That seems like a bug. The effect is that if a customer configures extraAnnotations, they expect them to be applied everywhere, but they will not be.